app.js

```diff
@@ -24,51 +24,29 @@ function imgProducts (path) {
 };
 
 function randomNumGen () {
-  var random = Math.floor((Math.random() * imgArray.length));
+  var random = Math.floor((Math.random() * productArray.length));
   return random;
 
```

Comment:
Unnecessary whitespace. And honestly you could just do this in one line like this:

```javascript
function randomNumGen(){
  return Math.floor((Math.random() * productArray.length))
}
```